Libraries/LibLine/Editor.cpp

```diff
@@ -921,6 +921,18 @@ void Editor::handle_read_event()
             }
             continue;
         }
+        // ^T
+        if (codepoint == ctrl('T')) {
+            if (m_cursor > 0 && m_buffer.size() >= 2) {
+                if (m_cursor < m_buffer.size())
+                    ++m_cursor;
+                u32 tmp = m_buffer[m_cursor - 1];
+                m_buffer[m_cursor - 1] = m_buffer[m_cursor - 2];
+                m_buffer[m_cursor - 2] = tmp;
```

Comment:
don't we have a `swap`?

Reply:
Oh, we do! I tried looking for it, but only in Libraries, because there's some gigantic json file in AK that matches most words that makes searching in AK difficult. (How do others deal with that?) Fixing...

Reply:
Done, better, thanks :)